tests/sdk/test_client.py

```
@@ -189,132 +186,967 @@ async def test_workspace_search(client_fixture: tuple[Honcho | AsyncHoncho, str]
         assert search_query in results[0].content
 
 
-@pytest.mark.asyncio
-async def test_get_deriver_status(client_fixture: tuple[Honcho | AsyncHoncho, str]):
+def test_client_init_with_api_key(client: TestClient):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_client_init_with_api_key`

Add return type annotation: `None`

(ANN201)